src/python/m5/ext/pystats/simstat.py

```diff
@@ -42,10 +42,10 @@ class SimStat(JsonSerializable):
     simulated_begin_time: Optional[Union[int, float]]
     simulated_end_time: Optional[Union[int, float]]
 
-    def __init__(self, creation_time: Optional[datetime],
-                 time_conversion: Optional[TimeConversion],
-                 simulated_begin_time: Optional[Union[int, float]],
-                 simulated_end_time: Optional[Union[int, float]],
+    def __init__(self, creation_time: Optional[datetime] = None,
```

Comment:
This should be a separate commit:
"python: Fix optional parameters with default None"

Reply:
Agreed.

Reply:
I can try to figure out how to do this

Reply:
resolved [here](https://github.com/darchr/gem5/compare/d974742952a4acfd9a5711f35d25cd42e84d0e70..bf98acaa9d8631db3a97f15eb4bd06524d690d68)